### PR TITLE
feat: prop added to new loan card to hide more cta in use component

### PR DESCRIPTION
**Pull request by roger-in-kiva**
`KivaClassicBasicLoanCardExp` component updated based in this requirements:

Let’s add the hover state back and remove the “more” cta for the next version of the test. 
Test group
category page, filter page, homepage include new loan card without the “more” (original design)

Control:
original /old loan cards in each of those loan card

Lending Home
by the time this ticket is done, we’ll have data on LH, so we can roll out the test and the loan card there can a/b more vs. no more 

Prop added to control lending home test in the future

### CODE REVIEW

File: src/components/LoanCards/KivaClassicBasicLoanCardExp.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		:style="{ minWidth: '230px', maxWidth: cardWidth }"
 	>
 		<div class="tw-grow">
-			<div>
+			<div :class="{ 'loan-card-active-hover': !allSharesReserved }">
```
Comment (dyersituations) [inline src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:9]: `allSharesReserved` doesn't exist on this component

Comment (roger-in-kiva) [reply src/components/LoanCards/KivaClassicBasicLoanCardExp.vue]: TY!

**Comment by dyersituations** [comment]
There is an issue now where if you hover the loan tags, the loan use is still underlined, but clicking the loan tags doesn't navigate. So we should ask product what the desired behavior is. Do we include the loan tags in the click target (and keep the underline when it's hovered), or do we not underline the loan use when the loan tags are hovered? Having the underline to suggest clickable state and not having the loan tags clickable is confusing to the user.
image

**Comment by roger-in-kiva** [comment]
> There is an issue now where if you hover the loan tags, the loan use is still underlined, but clicking the loan tags doesn't navigate. So we should ask product what the desired behavior is. Do we include the loan tags in the click target (and keep the underline when it's hovered), or do we not underline the loan use when the loan tags are hovered? Having the underline to suggest clickable state and not having the loan tags clickable is confusing to the user. image

Based on slack convo, loan tags should be clickable.

**Comment by dyersituations** [comment]
There are still gaps above and below the loan tags that show the loan use statement underlined but clicking doesn't navigate to the borrower profile. We probably want to wrap the image, loan tags, and loan use statement in a single router-link

File: src/components/LoanCards/KivaClassicBasicLoanCardExp.vue [hunk 2]
```diff
@@ -62,32 +62,38 @@
 				</div>
 
 				<!-- Loan tag -->
-				<loan-tag-v2 v-if="showTags && !isLoading" :loan="loan" :amount-left="amountLeft" />
+				<router-link
+					:to="customLoanDetails ? '' : `/lend/${loanId}`"
+					v-kv-track-event="['Lending', 'click-Read more', 'Photo', loanId]"
```
Comment (dyersituations) [inline src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:67]: Instead of `Photo`, this should be `Tag`

Comment (roger-in-kiva) [reply src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:67]: Thanks for the catch!

File: src/components/LoanCards/KivaClassicBasicLoanCardExp.vue [hunk 3]
```diff
@@ -62,32 +62,38 @@
...
+					class="tw-flex hover:tw-no-underline"
+				>
+					<loan-tag-v2 v-if="showTags && !isLoading" :loan="loan" :amount-left="amountLeft" />
```
Comment (dyersituations) [inline src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:70]: The loan tag will show an underline sometimes when clicked due to `focus`. I don't think we have `focus:tw-no-underline` available, so the parent `router-link` probably needs a new class and an entry added to the styles:

```
.loan-card-tag:focus {
  @apply tw-no-underline;
}
```

Comment (roger-in-kiva) [reply src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:70]: Looks like `focus:tw-no-underline` is working fine

**Review by dyersituations — APPROVED** [review]

**Pull request merged**